### PR TITLE
Rewards settings style

### PR DESCRIPTION
**Pull request by rossmoody**
Resolves: https://github.com/brave/brave-browser/issues/4035
Resolves: https://github.com/brave/brave-browser/issues/4171

Brave UI SHA update and a few string parity updates. 

Original Brave UI PR: https://github.com/brave/brave-ui/pull/442

### CODE REVIEW

File: components/brave_rewards/resources/ui/components/contributeBox.tsx [hunk 1]
```diff
@@ -120,7 +120,7 @@ class ContributeBox extends React.Component<Props, State> {
     }
 
     return (
-      <Grid columns={1} customStyle={{ maxWidth: '270px', margin: '0 auto' }}>
+      <Grid columns={1} customStyle={{ margin: '0 auto' }}>
```
Comment (rossmoody) [inline components/brave_rewards/resources/ui/components/contributeBox.tsx]: Trailing space

**Comment by rossmoody** [comment]
@NejcZdovc This one is ready for your review.

**Comment by jasonrsadler** [comment]
I noticed the text changed in `brave-ui`. Do we want to update here as well?
Screen Shot 2019-04-19 at 4 23 22 PM

**Comment by jasonrsadler** [comment]
Screen Shot 2019-04-19 at 4 29 20 PM
Text alignment for headers (at least for checkboxes) seems should be a little more to the left (based off spec photos from issue def)

**Comment by jasonrsadler** [comment]
Screen Shot 2019-04-19 at 4 33 20 PM
Not sure if part of PR but text underlaps bullet points (If NA to this PR, or if this is what we want, ignore)

**Comment by rossmoody** [comment]
@jasonrsadler  Just making sure, I think that screenshot you posted might be the "before" version. After should render more like this: https://d.pr/i/k8dSg2

I think we should do a followup for the wallet empty panel. From my gatherings, it looks like those updates would be back in Brave UI and might need a little more style help. I think maybe we'd be better off doing an ordered list instead of bullets with hard breaks. 

Had one remaining update that fixes the grant alerts not inheriting the width of the column.

**Comment by jasonrsadler** [comment]
Pulling and rebuilding and I'll double check.

👍 on wallet panel.

**Comment by jasonrsadler** [comment]
Yep. This is `rewards-settings` branch
Screen Shot 2019-04-19 at 6 39 28 PM

**Comment by jasonrsadler** [comment]
Screen Shot 2019-04-20 at 1 25 07 PM
👍

**Comment by NejcZdovc** [comment]
@rossmoody could you please check if grant notifications are the same. Text in ugp looks smaller then in ads grant

**Comment by rossmoody** [comment]
Good to go! This is them overlapping eachother. 

jpg

**Comment by karenkliu** [comment]
Can we use "Your earnings from ads are available." instead of "Your Ads earnings are available." so we match the grant messaging in the Your Wallet panel?

**Comment by rossmoody** [comment]
Good spot. This PR merged already but if you open an issue for this change I could take it on.

**Comment by karenkliu** [comment]
Should I open in brave-browser?

**Comment by rossmoody** [comment]
Yeah, [Brave Browser issues](https://github.com/brave/brave-browser/issues) would do the trick.

**Pull request merged**